screenshot_helper/screenshot_helper.ts

```diff
@@ -54,18 +54,19 @@ export async function compareScreenshot(data, golden, outputFolder = undefined):
       }
       if (!equal) {
         if (outputFolder) {
-          const diffPath = `${outputFolder}/diff-${path.basename(golden)}`;
-          console.log(diffPath);
```

Comment:
Oh yeah if you look at line 68, after the image is saved the promise rejects with the full path of the saved diff image. I would say unless it's for debugging purposes, we don't need to log the destination of diff image since it should be an input from the user.